Gemfile

```diff
@@ -15,5 +15,5 @@
 ruby '2.5.3'
 
 source 'https://rubygems.org/' do
-  gem 'kitchen-terraform', '~> 4.8'
+  gem 'kitchen-terraform', '~> 4.9'
```

Comment:
Why is this version change necessary?

Reply:
Yes it necessary, because only 4.9 compatible with terraform-0.12, please take a look https://rubygems.org/gems/kitchen-terraform/versions/4.8.1 and 
 https://rubygems.org/gems/kitchen-terraform/versions/4.9.0